acl-engine/src/test/groovy/javasabr/mqtt/acl/engine/model/condition/ConditionTest.groovy

```diff
@@ -16,17 +17,17 @@ class ConditionTest extends UnitSpecification implements ConditionMatcherAware {
     then:
         result == expectedResult
     where:
-        mqttUser                                  | condition                                                        | expectedResult
-        new TestMqttUser(null, "username2", null) | new AnyOfCondition(userNameEquals("username2"))                  | true
-        new TestMqttUser(null, "username2", null) | new AllOfCondition(userNameEquals("username2"))                  | true
-        new TestMqttUser(null, "username2", null) | new AnyOfCondition(userNameEquals("username1"))                  | false
-        new TestMqttUser(null, "username2", null) | new AllOfCondition(userNameEquals("username1"))                  | false
-        new TestMqttUser("username2", null, null) | clientIdEquals("username2")                                      | true
-        new TestMqttUser(null, null, "username2") | ipAddressEquals("username2")                                     | true
-        new TestMqttUser("username", null, null)  | clientIdEquals("username1")                                      | false
-        new TestMqttUser(null, null, "username")  | ipAddressEquals("username1")                                     | false
-        new TestMqttUser(null, null, null)        | new AnyUserCondition()                                           | true
-        new TestMqttUser(null, null, null)        | new AnyOfCondition(new UserNameCondition(new AnyValueMatcher())) | true
+        mqttUser                                  | condition                                                                | expectedResult
+        new TestMqttUser(null, "username2", null) | new AnyOfCondition(userNameEquals("username2"))                          | true
+        new TestMqttUser(null, "username2", null) | new AllOfCondition(userNameEquals("username2"))                          | true
+        new TestMqttUser(null, "username2", null) | new AnyOfCondition(userNameEquals("username1"))                          | false
+        new TestMqttUser(null, "username2", null) | new AllOfCondition(userNameEquals("username1"))                          | false
+        new TestMqttUser("username2", null, null) | clientIdEquals("username2")                                              | true
+        new TestMqttUser(null, null, "username2") | ipAddressEquals("username2")                                             | true
+        new TestMqttUser("username", null, null)  | clientIdEquals("username1")                                              | false
+        new TestMqttUser(null, null, "username")  | ipAddressEquals("username1")                                             | false
+        new TestMqttUser(null, null, null)        | new AnyUserCondition()                                                   | true
+        new TestMqttUser(null, null, null)        | new AnyOfCondition(new UserNameCondition(ValueMatcher.MATCH_ANY_STRING)) | true
```

Comment:
Can we consider adding on-demand static import for ValueMatcher here?

Reply:
@crazyrokr statim import reduce readability of the code, each time you need to check the import to understand what it's exactly